os/board/rtl8721csm/src/component/common/mbed/targets/hal/rtl8721d/serial_api.c

```diff
@@ -502,6 +502,13 @@ void serial_init(serial_t *obj, PinName tx, PinName rx)
 #endif
 }
 
+void serial_pin_init(PinName tx, PinName rx) {

```

Comment:
Please keep the same coding style in this file.
Other function does
```
void xxxx(yyyy)
{
```

Reply:
Thanks Done!~